packages/sui-ssr/server/utils/factory.js

```diff
@@ -68,6 +76,28 @@ export default ({path, fs, config: ssrConf = {}, assetsManifest}) => {
     }
   }
 
+  const useStaticsFolderByHost = expressStatic => {
+    let middlewares
+    if (isMultiSite) {
+      middlewares = multiSiteKeys.reduce((acc, hostPattern) => {
+        const site = siteByHostPattern(hostPattern)
+        if (acc[site]) return acc
+
+        return {
+          ...acc,
+          [site]: expressStatic(multiSiteStaticsFolder(site), EXPRESS_STATIC_CONFIG)
+        }
+      }, {})
+    }
+
+    return function serveStaticByHost(req, res, next) {
+      const site = siteByHost(req)
+      const middleware = isMultiSite ? middlewares[site] : expressStatic(DEFAULT_STATICS_FOLDER, EXPRESS_STATIC_CONFIG)
```

Comment:
We should keep the same default functionality as before.

```suggestion
      const middleware = isMultiSite ? middlewares[site] : expressStatic(DEFAULT_STATICS_FOLDER)
```